src/Core/Grid/Data/Factory/CategoryDataFactory.php

```diff
@@ -26,57 +26,9 @@
 
 namespace PrestaShop\PrestaShop\Core\Grid\Data\Factory;
 
-use PrestaShop\PrestaShop\Core\Grid\Data\GridData;
-use PrestaShop\PrestaShop\Core\Grid\Record\RecordCollection;
-use PrestaShop\PrestaShop\Core\Grid\Search\SearchCriteriaInterface;
-
 /**
  * Class CategoryDataFactory decorates DoctrineGridDataFactory configured for categories to modify category records.
  */
-final class CategoryDataFactory implements GridDataFactoryInterface
+class CategoryDataFactory extends AbstractCategoryDataFactory
```

Comment:
What is the reason behind removing the final keyword?

Reply:
Oups ! Done